docker-compose.yml

```diff
@@ -0,0 +1,174 @@
+version: '3.7'
+services:
+  provisioning-db:
+    restart: on-failure
+    image: docker.io/postgres:13.1
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "postgres"]
+      interval: 2s
+      timeout: 3s
+      retries: 10
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=provisioning
+    volumes:
+      - ./compose-data/postgres/backend:/var/lib/postgresql/data
+    expose:
+      - 5432
+    
+  redis:
+    image: registry.redhat.io/rhel8/redis-6
+    healthcheck:
+      test: [ "CMD", "redis-cli", "ping" ]
+      interval: 2s
+      timeout: 2s
+      retries: 10
+    expose:
+      - 6379
+    volumes:
+      - ./compose-data/redis:/data
```

Comment:
This needs to login, can you fallback to upstream? Also add appropriate tags, this works for me:

```yaml
  redis:
    profiles:
      - "redis"
      - "sources"
      - "backend"
    image: docker.io/redis:latest
    healthcheck:
      test: [ "CMD", "redis-cli", "ping" ]
      interval: 2s
      timeout: 2s
      retries: 10
    expose:
      - 6379
    volumes:
      - ./data/redis:/data
```